lib/amazon-pricing/helpers/instance-type.rb

```diff
@@ -195,7 +195,12 @@ def self.api_name_to_nf(name)
             type = sizes[-2].split('.').last      # 'metal' already largest, so 2nd largest
           end
         end
-        size_to_nf[type]
+        full_type = type.gsub(/xl$/, 'xlarge')
+        nf = size_to_nf[full_type]
+        if nf.nil?
+          raise "No normalization factor found for api name: #{name}"
```

Comment:
I thought we were going to wait to do this until we had moved this into CHT only repo?

Reply:
Yep, sorry, wasn't thinking. Took it out.